wagtail/users/migrations/0007_userprofile_avatar_choice.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2017-11-24 14:18
+from __future__ import unicode_literals
```

Comment:
It seems to me that we don't really need `__future__` imports anymore: we are going to drop python 2 support.